RadixWallet/Clients/SecurityCenterClient/SecurityCenterClient+Interface.swift

```diff
@@ -153,6 +153,7 @@ enum SecurityProblem: Hashable, Sendable, Identifiable {
 	}
 
 	enum ProblemType: Hashable, Sendable, CaseIterable {
+		case securityShields
```

Comment:
Note: this will be moved into Sargon https://github.com/radixdlt/sargon/pull/287